snapcraft/plugins/python2.py

```diff
@@ -206,18 +206,11 @@ def build(self):
     def snap_fileset(self):
         fileset = super().snap_fileset()
         fileset.append('-usr/bin/pip*')
-        fileset.append('-usr/lib/python*/dist-packages/easy-install.pth')
-        fileset.append('-usr/lib/python*/dist-packages/__pycache__/*.pyc')
-        fileset.append('-usr/lib/python*/*.pyc')
-        fileset.append('-usr/lib/python*/*/*.pyc')
-        fileset.append('-usr/lib/python*/*/*/*.pyc')
-        fileset.append('-usr/lib/python*/*/*/*/*.pyc')
-        fileset.append('-usr/lib/python*/*/*/*/*/*.pyc')
-        fileset.append('-usr/lib/python*/*/*/*/*/*/*.pyc')
-        fileset.append('-usr/lib/python*/*/*/*/*/*/*/*.pyc')
-        fileset.append('-usr/lib/python*/*/*/*/*/*/*/*/*.pyc')
-        fileset.append('-usr/lib/python*/*/*/*/*/*/*/*/*/*.pyc')
-        fileset.append('-usr/lib/python*/*/*/*/*/*/*/*/*/*/*.pyc')
+        # .pth files are only read from the built-in site-packages directory.
+        # We use PYTHONPATH for everything so not needed.
+        fileset.append('-**/*.pth')
+        # This is a major cause of inter part conflict.
+        fileset.append('-**/*.pyc')
```

Comment:
This actually pleases me _more_ than the previous PR, as I don't feel like we're fighting snapcraft here.